### PR TITLE
taxonomy: translations for shapes and materials

### PR DESCRIPTION
**Pull request by teolemon**
### What
taxonomy: translations for shapes and materials

### CODE REVIEW

**Comment by teolemon** [comment]
ERROR - pt:capsula-de-cafe already is associated to en:coffee-capsule (packaging_shapes) - pt:capsula-de-cafe cannot be mapped to entry en:coffee-pad
[Fri Dec  8 13:49:01 2023] build_lang.pl: Errors in the packaging_shapes taxonomy definition at /opt/product-opener/lib/ProductOpener/Tags.pm line 1496.
[Fri Dec  8 13:49:01 2023] build_lang.pl: Compilation failed in require at /opt/product-opener/scripts/build_lang.pl line 32.
[Fri Dec  8 13:49:01 2023] build_lang.pl: BEGIN failed--compilation aborted at /opt/product-opener/scripts/build_lang.pl line 32.
<h1>Software error:</h1>
<pre>Errors in the packaging_shapes taxonomy definition at /opt/product-opener/lib/ProductOpener/Tags.pm line 1496.

File: taxonomies/packaging_materials.txt [hunk 1]
```diff
@@ -1540,6 +1698,53 @@ description:pt:A cortiça é um material leve e elástico feito da casca do sobr
...
+hu:Textil, textíliák
+id:Tekstil, tekstil
+io:Tekstilo, tekstili
+it:Tessile, tessili
+lt:Tekstilė, tekstilės
+lv:Tekstils, tekstilizstrādājumi
+mk:Текстил, текстилни материјали
+mt:Tekstil, tekstili
+nb:Tekstil, tekstiler
+nl:Textiel, textiel
```
Comment (teolemon) [inline taxonomies/packaging_materials.txt:1731]:
```suggestion
nl:Textiel
```

File: taxonomies/packaging_materials.txt [hunk 2]
```diff
@@ -173,6 +242,11 @@ hu:Újrahasznosított anyag, Újrahasznosított anyagból
 it:Materiale riciclato
 nl:Hergebruikt materiaal
 pt:Material reciclado
+es:Material reciclado
+ja:リサイクル素材
+ko:재활용 재료
+ru:Переработанный материал
+zh:可回收材料
```
Comment (teolemon) [inline taxonomies/packaging_materials.txt:249]:
```suggestion
```

File: taxonomies/packaging_materials.txt [hunk 3]
```diff
@@ -134,6 +156,32 @@ hu:Újrahasznosítható anyag
...
+ro:Material reciclabil
+ru:Перерабатываемый материал
+sk:Recyklovateľný materiál
+sl:Reciklabilni material
+sq:Materiali i riciklueshëm
+sr:Материјал за рециклирање
+sv:Återvinningsbart material
+tr:Geri dönüştürülebilir malzeme
+uk:Перероблюваний матеріал
+zh:可回收材料
```
Comment (teolemon) [inline taxonomies/packaging_materials.txt:184]:
```suggestion
```

**Review by benbenben2 — APPROVED** [review]
Looks good to me

**Review by stephanegigandet — APPROVED** [review]
Thanks

**Pull request merged**